docs/update-from-client-side-helpers-to-server-side.md

```diff
@@ -0,0 +1,229 @@
+# Upgrading your Client-side integration to a Server-side integration
+
+We recommend using a combination of client-side and server-side code to integrate the PayPal Buttons on your e-commerce website. If you currently have client-side only PayPal Buttons integration, this page will describe how to update your integration to the recommended approach.
+<br/><br/>
+> 
+> **Know before you code**
+>  
+> [How to Setup a Developer Account](https://www.youtube.com/watch?v=O_9G722SpXQ&t=72s)
+> <br/><br/>
+
+# What do I need to change?
+
+If you are reading this page, we expect that you already have a client-side PayPal Button integration in place.  That said, several steps are required to get you up and running with the recommended approach. 
+
+Specifically:
+
+1. Setup a server-side integration with the [PayPal REST API's](https://developer.paypal.com/api/rest/).
+
+1. `createOrder` JavaScript callback must be changed to use your server-side to create and return an Order ID using the PayPal backend API's.
+
+1. `onApprove` JavaScript callback must be changed to use your server-side to complete transactions using the PayPal backend API's.
+
+<br/><br/>
+
+# 1. Setup a server-side integration with the PayPal REST API's
+
+PayPal REST API's follow a common integration approach used by other RESTful API's out there in the world.  For PayPal, you will minimally need to do the following:
+
+1. Setup a PayPal developer account if you have not already.
+
+1. Create a PayPal REST application and obtain a client + secret
+
+1. Use the client/secret combination to obtain an oAuth access token
+
+1. Use the oAuth access token to directly make REST calls to the PayPal Order API's -- which you will need for this document.
+
+<br/><br/>
+
+# 2. Change your `createOrder()` callback
+
+Your current integration may likely be using the client-side order create functions.  You will need to replace this code to call your server side instead.  Examples on before and after are below.
+
+## Current state of client-side integration behavior
+
+With the client-side integration pattern, the SDK code would take care of calling the v2 Orders API on your behalf. It would also handle the authorization for you.
+
+```js
+createOrder(data, actions) {
```

Comment:
I wonder if we can use that diff feature of github here where you do the three backticks and diff like so:

```diff
function addTwoNumbers (num1, num2) {
-  return 1 + 2
+  return num1 + num2
}
```

Thoughts?

Reply:
I do like the diff code snippets too. We originally leveraged this diff feature but ended up removing it so we could have descriptive text before each snippet.

Reply:
Ah that makes sense. My overall concern then would be calling out this is the bad pattern. How about we do a `**Previous Code**` on the line above the snippet. Or something to painfully call out this is the legacy code snippet? I always get nervous about those fast copy pasters